aesara/link/c/cmodule.py

```diff
@@ -2717,9 +2719,20 @@ def default_blas_ldflags():
 
         # We need to catch warnings as in some cases NumPy print
         # stuff that we don't want the user to see.
-        # I'm not able to remove all printed stuff
-        with warnings.catch_warnings(record=True):
-            numpy.distutils.system_info.system_info.verbosity = 0
+        @contextmanager
+        def catch_numpy_warnings():
+            with warnings.catch_warnings(record=True):
+                numpy.distutils.system_info.system_info.verbosity = 0  # side-effect
+                sio = io.StringIO()
+                with redirect_stderr(sio):
```

Comment:
Are we sure that numpy is printing to stderr and not to stdout?

Reply:
I'm pretty sure, but not 100%. That's why I need someone with Windows to test this! :joy:

Reply:
Indeed, see https://github.com/aesara-devs/aesara/pull/980#issuecomment-1152783194.

Now fixed by filtering both stdout and stderr.